src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/Cas2ApplicationSummaryEntity.kt

```diff
@@ -0,0 +1,115 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity
+
+import org.springframework.data.domain.Page
+import org.springframework.data.domain.Pageable
+import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.stereotype.Repository
+import java.time.LocalDate
+import java.time.OffsetDateTime
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.Id
+import javax.persistence.Table
+
+@Repository
+interface ApplicationSummaryRepository : JpaRepository<Cas2ApplicationSummaryEntity, String> {
+//  @Query(countQuery =
+//    """
+//    SELECT COUNT(*)
+//      FROM cas_2_applications a
+//    WHERE a.created_by_user_id = :userId
+//    """,
+//    nativeQuery = true,
+//  )
+  fun findByUserId(userId: String, pageable: Pageable?): Page<Cas2ApplicationSummaryEntity>
+
+//  @Query(countQuery =
```

Comment:
Left in count queries in case these are needed to fix failing integration tests.

Reply:
Issue with integration tests resolved and all are now passing.